pgactivity/queries/get_data_directory.sql

```diff
@@ -0,0 +1,2 @@
+-- Get the path of the pidfile
+SHOW data_directory;
```

Comment:
`SHOW` would not work when we'll move to using psycopg 3. Perhaps we can keep the query to `pg_settings`?

Reply:
I like this because we have an error.
The query returns nothing when we don't have the privilege.

I can revert to the previous query.

Reply:
Sorry, I'm wrong actually. I mixed up `SHOW` and `SET`. `SHOW` should work.